lib/connectors.ts

```diff
@@ -18,9 +19,17 @@ type Connector = {
   seasonOneProxy?: Address | null;
   devProxy?: Address | null;
   testDevProxy?: Address | null;
-}
+  easBaseUrl?: string;
+  easAttestationContract?: Address | null;
+  scoutgameEASResolver?: Address | null;
+  scoutgameErc20TokenDev?: Address | null;
+  scoutgameScoutProtocolProxyDev?: Address | null;
```

Comment:
These are the 3 main keys

I've been considering refactoring these under a dev | prod key with same type to avoid duplication of keys